subnet-topos.yml

```
@@ -18,121 +18,185 @@ services:
     networks:
       - local-erc20-messaging-infra-docker
 
+  # Install the edge binary in the current volume
+  install-edge:
+    image: ghcr.io/topos-protocol/topos:${TOPOS_VERSION}
+    container_name: install-edge
+    command:
+      - "setup"
+      - "subnet"
+      - "--path"
+      - "/data/"
+      - "--release"
+      - "v${TOPOS_EDGE_VERSION}"
+    environment:
+      - TOPOS_EDGE_VERSION=${TOPOS_EDGE_VERSION}
+    volumes:
+      - topos-data:/data
+
+  topos-init:
+    container_name: topos-init
+    command: sh -c "mkdir -p /data/node/ -p /data/subnet/topos && \
+      mv /data/data-* /data/node/ && \
+      mv /data/genesis.json /data/subnet/topos/genesis.json && \
+      chmod a+rwx /data/* && ls -lRa /data && cat /data/subnet/topos/genesis.json"
+    image: debian:bullseye-slim
+    depends_on:
+      keystore-init:
+        condition: service_completed_successfully
+      install-edge:
+        condition: service_completed_successfully
+    volumes:
+      - topos-data:/data
+    healthcheck:
+      test: ["CMD-SHELL", "test -f /data/subnet/topos/genesis.json"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+      start_period: 5s
+    networks:
+      - local-erc20-messaging-infra-docker
+
   topos-node-1:
-    image: ghcr.io/topos-protocol/polygon-edge:${TOPOS_EDGE_VERSION}
+    image: ghcr.io/topos-protocol/topos:${TOPOS_VERSION}
     container_name: topos-node-1
     init: true
     command:
-      - "server"
-      - "--data-dir"
-      - "/data/data-1"
-      - "--chain"
-      - "/data/genesis.json"
-      - "--grpc-address"
-      - "0.0.0.0:9632"
-      - "--libp2p"
-      - "0.0.0.0:1478"
-      - "--jsonrpc"
-      - "0.0.0.0:8545"
-      - "--price-limit"
-      - ${PRICE_LIMIT}
-      - "--seal"
+      - "node"
+      - "--edge-path"
+      - "/data"
+      - "up"
+      - "--name"
+      - "data-1"
```

```suggestion
      - "node-1"
```

we should either rename this container as `boot` or `node-1`